pkg/controllers/dynakube/otelc/endpoint/reconciler.go

```diff
@@ -99,17 +98,9 @@ func (r *Reconciler) reconcileConfigMap(ctx context.Context) error {
 func (r *Reconciler) generateData() (map[string]string, error) {
 	data := make(map[string]string)
 
-	dtEndpoint := r.dk.APIURL() + "/v2/otlp"
-
-	if r.dk.ActiveGate().IsEnabled() {
-		tenantUUID, err := r.dk.TenantUUID()
-		if err != nil {
-			return data, err
-		}
-
-		serviceFQDN := capability.BuildServiceName(r.dk.Name) + "." + r.dk.Namespace + ".svc"
-
-		dtEndpoint = fmt.Sprintf("https://%s/e/%s/api/v2/otlp", serviceFQDN, tenantUUID)
+	dtEndpoint, err := otlp.GetOtlpIngestEndpoint(r.dk)
```

Comment:
since package already has otlp, it's a best practice to simplify the name of the function.
```
otlp.GetIngestEndpoint(r.dk)
```